### ISSUE TITLE
Accessibility: Left menu should be navigation

### ISSUE THREAD

**Issue by GuySartorelli** [silverstripe/silverstripe-admin#1971]
The left menu should either be wrapped in a semantic `<nav>` element, or (to preserve existing selectors for CSS and JS) should have[ `role="navigation"`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Roles/navigation_role) applied.

See https://www.w3.org/WAI/ARIA/apg/practices/landmark-regions/ to learn more about navigation and other landmarks

## Acceptance criteria

- [ ] The left menu is identified as a "navigation" landmark and has an appropriate label

## PRs
- https://github.com/silverstripe/silverstripe-admin/pull/1987
- https://github.com/silverstripe/developer-docs/pull/807

**Comment by GuySartorelli** [comment]
PRs merged
